specification/hybriddatamanager/resource-manager/readme.md

```diff
@@ -39,8 +39,7 @@ input-file:
 - Microsoft.HybridData/stable/2016-06-01/hybriddata.json
 directive:
   - suppress:
-    - R2016 #to suppress (PatchBodyParametersSchema/R2016/RPCViolation)
-    - R2062 #to suppress (XmsResourceInPutResponse/R2062/RPCViolation)
+    - R1001 #to suppress (OperationIdNounVerb/R1001/SDKViolation)
```

Comment:
Why did you delete the suppressions for R2016 and R2062? They are failing in lintdiff now.

Reply:
The suppressions were added by mistake without approval from the review team. Once we get the approval for the suppressions, I will add the required directives again.

Reply:
Suppression added